pkg/route/api/v1/types.go

```diff
@@ -43,6 +43,10 @@ type RouteSpec struct {
 	// be defaulted to Service.
 	To kapi.ObjectReference `json:"to"`
 
+	// AdditionalTos is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	AdditionalTos []kapi.ObjectReference `json:"additionalTos,omitempty"`
```

Comment:
> RouteTargetReference

How about WeightedObjectReference? I am thinking I could reuse this for A/B deployments where we would have references to pod templates (assuming we ever start taking references to pod templates)